### PR TITLE
ACMS-1389: Content type templates for Article, Person, Place, Event, Page

### PR DESCRIPTION
**Pull request by rajeshreeputra**
**Motivation**
Fixes #ACMS-1389

**Proposed changes**
- Rewrite Image field to include DAM media.
- Updated content type templates for Article, Person, Place, Event, Page for Media(CORE, DAM).
- Added new view mode 'Referenced Image' for Place and Person content type.
- Updated Article, Event Site studio templates, default view mode.
- Added update hook to update view mode for Article, Event, Person, Place.

**Alternatives considered**
NA

**Testing steps**
- Use this branch
- Install site using acms site:install
- Install ACMS DAM module drush in acquia_cms_dam
- Login into application
- Create node using any content type
- Select DAM media for image in first node
- Select CORE media for image in second node
- Verify images rendering fine on both nodes

**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [ ] Manual testing by a reviewer

### CODE REVIEW

**Review by apathak18 — CHANGES_REQUESTED** [review]
Do we a composer.lock file in the PR? @rajeshreeputra

File: modules/acquia_cms_article/acquia_cms_article.install [hunk 1]
```diff
@@ -82,3 +82,70 @@ function acquia_cms_article_update_8003() {
     }
   }
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Update Article display modes.
+ */
+function acquia_cms_article_update_8004() {
```
Comment (apathak18) [inline modules/acquia_cms_article/acquia_cms_article.install:91]: Can we create a service class for hook_update logic? and call that service class where we need? @rajeshreeputra 

As I can notice that every content_type.install hook_update having same logic with different variable names.

cc: @vishalkhode1

**Review by apathak18 — APPROVED** [review]
LGTM

**Review by rajeshreeputra — COMMENTED** [review]
All changes required for ACMS-1389 and ACMS-1503 has been completed, hence moving ahead for review.

**Review by vishalkhode1 — APPROVED** [review]

**Pull request merged**